app/boards/shields/nice_view/Kconfig.defconfig

```diff
@@ -26,4 +25,24 @@ endchoice
 
 endif # ZMK_DISPLAY
 
+config CUSTOM_WIDGET_STATUS
+    bool "Custom status widget"
+    default y if ZMK_DISPLAY_STATUS_SCREEN_CUSTOM
```

Comment:
I think maybe instead enhance the existing custom screen config to `select` this symbol?

Reply:
So, you could instead do something like:

```
config ZMK_DISPLAY_STATUS_SCREEN_CUSTOM
  imply CUSTOM_WIDGET_STATUS
  imply CUSTOM_WIDGET_BLAH
```

Doing so aligns the "hierarchy" of the Kconfig matches the code. The custom screen "consumes" or "uses" the widgets, so I think it works better to have the Kconfig for that custom screen implying/enabling the widgets it uses.